app/kafka/transformer_topic_consumer.rb

```diff
@@ -5,31 +5,35 @@ def initialize
     Thread.new { subscribe }
   end
 
+  def avro
```

Comment:
you can remove this function, it's not used

Reply:
Removed